### ISSUE TITLE
Boost Carousel flipping backward

### ISSUE THREAD

**Issue by timbray** [cheeaun/phanpy#287]
**Describe the bug**
Sometime in the past few days, the Boost carousel does this:  I leave my mouse over the right arrow and tap-tap-tap my way through the boosted posts, carousel clicks left with each tap, but then sometimes (it's inconsistent) the carousel jumps way back to the right, to the first boost).  It seems that leaving more time between taps minimizes (but doesn't eliminate) this behavior.

- Which site: [e.g. dev.phanpy.social OR phanpy.social]
phanpy.social
- Which instance: [e.g. mastodon.social]
cosocial.ca

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] MacOS
 - Browser [e.g. chrome, safari]: Safari
 - Version [e.g. 22]: Version 17.0 (19616.1.27.211.1)

**Smartphone (please complete the following information):**
[not observed on my Pixel]

**Comment by timbray** [comment]
And just now, after I filed the issue, the carousel got stuck, would go forward then back on each click.

I could make a little screencast if that would help?

**Comment by cheeaun** [comment]
@timbray a screen recording would be useful 🙇‍♂️

**Comment by timbray** [comment]
Here you go: https://www.tbray.org/tmp/phan.mov - sorry, kind of big

**Comment by timbray** [comment]
OK, phanpy went completely blank-screen crazy and I couldn't convince it to show me everything. This is on Safari. So I went and wiped all its cookies and re-logged-in, now it's fine again.  So while I suspect there is a weird bug lurking deep somewhere, maybe not a very urgent issue.

**Comment by cheeaun** [comment]
Hmm, I'm able to reproduce this erratic scrolling on Safari 17.1 too. Seems like a browser bug as it doesn't happen on Chrome and Firefox — it seems that they auto-debounce the continuous taps, but Safari kinda go haywire with the scroll position and jumps to a weird position instead 😅😩

@timbray does this bug happen recently, maybe after upgrading macOS + Safari? Was it working fine before in older Safari (v16 maybe)?

**Comment by timbray** [comment]
Yes, definitely a recent arrival but sorry, can't put a precise date on it. Maybe the latest MacOS? But it went away for me when I cleared phanpy cookies.
